conversation_reconstruction_dataflow/ingest_revisions/dataflow_main.py

```diff
@@ -68,19 +71,36 @@ def run(known_args, pipeline_args):
   with beam.Pipeline(options=pipeline_options) as p:
     pcoll = (p | ReadFromText(known_args.input)
                    | beam.ParDo(WriteDecompressedFile())
-                   | beam.io.Write(bigquery.BigQuerySink(known_args.table, schema=known_args.schema)))
+                   | beam.io.Write(bigquery.BigQuerySink(known_args.table, schema=known_args.schema, validate = True)))
 
 def truncate_content(s):
     dic = json.loads(s) 
```

Comment:
Add definition documentation string to say what the argument "s" is, and the expected shape of the returned object. This function is worth writing a test for also.

Reply:
Added test for truncating and timeout